packages/graphql/app.js

```diff
@@ -29,6 +30,28 @@ app.use((err, req, res, next) => {
   res.status(500).send("Something broke!");
 });
 
+// Setup pdf service
+app.get("/graphql/pdfr/:slug", async (req, res) => {
+  const browser = await puppeteer.launch();
+  const page = await browser.newPage();
+  const pdfFrontend =
+    config.pdfFrontend || `${req.protocol}://${req.headers.host}`;
+  const url = `${pdfFrontend}/${
+    req.params.slug
+  }/vragen-en-uitkomst?data=${encodeURIComponent(req.query.data)}`;
```

Comment:
Can we make the page `vragen-en-uitkomst` dynamic in case we change this. URI's have been changed frequently so maybe we can import this?